app/rules/v2_to_v3/question_conversions.py

```diff
@@ -0,0 +1,90 @@
+import re
+
+from app.helpers.rule_processor import process_rules
+from app.rules.v2_to_v3.general_conversions import (
+    delete_parent_id,
+    update_guidance_content_key,
+    rename_content_key,
+)
+
+
+def question_conversions(all_questions):
+
+    question_rules = {
+        "rename_single_questions_key": rename_single_questions_key,
+        "delete_parent_id": delete_parent_id,
+        "update_guidance_content_key": update_guidance_content_key,
+        "rename_question_definitions_content_key": rename_question_definitions_content_key,
+        "update_placeholder_in_question_titles": update_placeholder_in_question_titles,
+    }
+
+    process_rules(all_questions, question_rules)
+
+
+def update_placeholder_in_question_titles(question):
+
+    if "title" in question:
+        placeholders = []
+        question_title = question["title"]
```

Comment:
```suggestion
try:
    question_title = question["title"]
except KeyError:
    return None
```

Reply:
Make this the first call so it returns before the placeholders list is instantiated if necessary